### DISCUSSION TITLE
Pass mp3 filename to the status-json.xsl

### DISCUSSION THREAD

**Discussion by carrzkiss** [savonet/liquidsoap#1754] [Q&A]
Hello, All.

I was wondering if it is possible to pass the filename of the audio which is playing to the status-json.xsl?
This would work out great when processing the data from the stream to match what is in our database.
I can match the "**Artist - Song Title**" to the database if it is not possible.
Though 99.9% of the time, there will only be one instance of the values in the database. However, there is always the chance that there is a second or third instance.
For example.
We have the 
Article - Album - Song(s) - filename
KISS - Destroyer - Detroit Rock City - 9872978234.mp3
KISS - Double Platinum - Detroit Rock City - 787359835.mp3

So, in the case of the two songs above, they are going to have the same "Artist - Song Title."

**Comment by StudentSA** [comment]
Im assuming you are referring to icecast when talking about 'status-json.xsl'. The title field shown there is populated from the mp3 metadata tags "artist - title".

A quick solution might be to update your mp3 files metadata and set either artist or title as the filename?

**Comment by carrzkiss** [comment]
Thanks, StudentSA.
Unfortunately I cannot do that.

So there is no way to add in more tags to the Icecast?

**Comment by StudentSA** [comment]
You might find this useful:

https://www.liquidsoap.info/doc-1.4.3/icy_metadata.html
